third_party/2and3/requests/packages/urllib3/__init__.pyi

```diff
@@ -1,8 +1,8 @@
 from typing import Any
 from . import connectionpool
-from . import filepost
-from . import poolmanager
-from . import response
+from . import filepost as filepost
```

Comment:
Oh, good catch. This is covered by a different notion, where submodules of packages are made available as attributes of their parent package as a side-effect of importing the submodule (implicitly or explicitly). So the 'as' idiom should not be needed here. But yeah, a PEP clarification would be good.